modules/javafx.graphics/src/main/native-glass/mac/GlassMenu.m

```diff
@@ -245,6 +244,8 @@ - (void)_setShortcut:(jchar)jshortcut modifiers:(jint)jmodifiers
                    jshortcut <= com_sun_glass_events_KeyEvent_VK_F24) {
             int delta = jshortcut - com_sun_glass_events_KeyEvent_VK_F13;
             shortcut = [NSString stringWithFormat:@"%C", (unsigned short)(NSF13FunctionKey + delta)];
+        } else {
```

Comment:
This else clause can be removed. JavaFX should never set the NSFunctionKeyMask so this PR can be just a one-line change (remove line 239).

Sorry I wasn't clearer about this earlier. I was basically diagnosing a different bug in the same area of code. Not your concern.

Reply:
A-ha, now I understand, thanks for the clarification!